ledger/acctupdates.go

```diff
@@ -445,6 +451,12 @@ func (au *accountUpdates) produceCommittingTask(committedRound basics.Round, dbR
 	// calculate the number of pending deltas
 	dcr.pendingDeltas = au.deltasAccum[offset] - au.deltasAccum[0]
 
+	proto := config.Consensus[au.versions[offset]]
```

Comment:
shouldn't this be in catchpoint tracker or at least in online accounts tracker?

Reply:
yes, probably, the only reason I did it this way account updates "drives" the commit. In fact, this logic needs to be moved to tracker.go